lnbits/core/views/api.py

```diff
@@ -236,10 +236,11 @@ async def api_payments_create_invoice(data: CreateInvoiceData, wallet: Wallet):
             webhook=data.webhook,
             internal=data.internal,
         )
-    except InvoiceFailure as e:
-        raise HTTPException(status_code=520, detail=str(e))
-    except Exception as exc:
-        raise exc
+    except InvoiceFailure as exc:
```

Comment:
```suggestion
    except Exception as exc:
```

Otherwise all other exceptions will leak to the client.